### PR TITLE
Add correct ukrainian l10n

### PR DESCRIPTION
**Pull request by alexmazaltov**

### CODE REVIEW

**Comment by alexmazaltov** [comment]
> Hey! undefined has not signed the Canonical CLA which is required to get this contribution merged on this project.
> Please head over to https://ubuntu.com/legal/contributors to read more about it.

Sorry, i didn't know that CLA needs to be signed. So I signed it already:
image

**Comment by serhii-dolia** [comment]
As a Ukrainian speaker I can agree that there's no profanity or hate speech in this translation.

**Comment by inobelar** [comment]
@alexmazaltov please note that your edits duplicate an already existing pull request (PR) https://github.com/canonical/ubuntu-desktop-provision/pull/171, and also that adding translation file from your PR is not enough - to activate the lines, it seems that you need to make changes to `packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations.dart`.

I propose to join forces to clean up vandalism there :)

**Comment by alexmazaltov** [comment]
> @alexmazaltov please note that your edits duplicate an already existing pull request (PR) #171, and also that adding translation file from your PR is not enough - to activate the lines, it seems that you need to make changes to `packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations.dart`.
> 
> I propose to join forces to clean up vandalism there :)

Thank you for pointing the proper PR. i also found few lines with hate translations in file `/lib/l10n/ubuntu_bootstrap_localizations_uk.dart` 
And i've migrated the stuff from that PR you pointed.

**Comment by olegchigirin** [comment]
As a Ukrainian speaker, I confirm that nothing rude or irrelevant here.

**Comment by webknjaz** [comment]
Another +1 from a native Ukrainian speaker.

**Comment by zen-tools** [comment]
Can confirm, the diff looks good.

**Comment by AndrewDryga** [comment]
Native speaker here, the diff looks good.

**Review by delgod — APPROVED** [review]

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+  "welcomeRepairDescription": "Під час виправлення буде перевстановлено все програмне забезпечення, не торкаючись документів чи налаштувань.",
+  "@welcomeRepairDescription": {},
+  "welcomeTryOption": "Спробувати {RELEASE}",
+  "@welcomeTryOption": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
+  "welcomeTryDescription": "Ви можете спробувати {RELEASE}, не роблячи ніяких змін на вашому комп'ютері.",
```
Comment (olegchigirin) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb:29]:
```suggestion
  "welcomeTryDescription": "Ви можете спробувати {RELEASE}, не роблячи жодних змін на вашому комп'ютері.",
```

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+  },
+  "rstTitle": "Вимкнути RST",
+  "@rstTitle": {},
+  "rstDescription": "Цей комп’ютер використовує технологію Intel RST (Rapid Storage Technology). Перед встановленням Ubuntu вам потрібно вимкнути RST у Windows.",
+  "@rstDescription": {},
+  "quitButtonText": "Скасувати встановлення",
+  "@quitButtonText": {},
+  "configureSecureBootTitle": "Налаштувати безпечне завантаження",
+  "@configureSecureBootTitle": {},
+  "configureSecureBootDescription": "Ви вирішили встановити стороннє програмне забезпечення для драйверів. Для цього необхідно вимкнути безпечне завантаження.\nДля цього потрібно вибрати ключ безпеки зараз, та ввести його після перезавантаженні системи.",
```
Comment (olegchigirin) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb:65]:
```suggestion
  "configureSecureBootDescription": "Ви вирішили встановити стороннє програмне забезпечення для драйверів. Для цього необхідно вимкнути безпечне завантаження.\nДля цього потрібно обрати ключ безпеки зараз, та ввести його після перезавантаженні системи.",
```

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+  "dontInstallDriverSoftwareNow": "Наразі не встановлювати програмне забезпечення драйвера",
+  "@dontInstallDriverSoftwareNow": {},
+  "rstInstructions": "Для отримання інструкцій відкрийте цю сторінку на телефоні або іншому пристрої: <a href=\"https://{url}\">{url}</a>",
+  "@rstInstructions": {
+    "type": "text",
+    "placeholders": {
+      "url": {}
+    }
+  },
+  "dontInstallDriverSoftwareNowDescription": "Ви можете встановити його пізніше з розділу \"Програми та оновлення\".",
```
Comment (olegchigirin) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb:82]:
```suggestion
  "dontInstallDriverSoftwareNowDescription": "Ви можете встановити його пізніше у розділі \"Програми та оновлення\".",
```

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+  },
+  "confirmPageTitle": "Записати зміни на диск",
+  "@confirmPageTitle": {},
+  "shutdown": "Вимкнути",
+  "@shutdown": {},
+  "restartIntoWindows": "Перезавантажити в Windows",
+  "@restartIntoWindows": {},
+  "installDriversTitle": "Встановити стороннє програмне забезпечення для графіки та обладнання Wi-Fi",
+  "@installDriversTitle": {},
+  "installDriversSubtitle": "Ці драйвери підпадають під дію ліцензійних умов, включених до їх документації. Вони є пропрієтарними..",
```
Comment (olegchigirin) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb:169]:
```suggestion
  "installDriversSubtitle": "Ці драйвери підпадають під дію ліцензійних умов, включених до їхньої документації. Вони є пропрієтарними..",
```

**Review by olegchigirin — COMMENTED** [review]
Small corrections to localization

**Review by taurus-forever — APPROVED** [review]

**Comment by spydon** [comment]
The new weblate instance (with validators) is now up and all translations are done through there.

**Pull request closed without merging**